src/main/java/de/tum/cit/aet/artemis/iris/service/settings/IrisSubSettingsService.java

```diff
@@ -415,7 +424,9 @@ public IrisCombinedTextExerciseChatSubSettingsDTO combineTextExerciseChatSetting
     public IrisCombinedLectureChatSubSettingsDTO combineLectureChatSettings(ArrayList<IrisSettings> settingsList, boolean minimal) {
         var enabled = getCombinedEnabled(settingsList, IrisSettings::getIrisLectureChatSettings);
         var rateLimit = getCombinedRateLimit(settingsList);
-        return new IrisCombinedLectureChatSubSettingsDTO(enabled, rateLimit, null);
+        var allowedVariants = !minimal ? getCombinedAllowedVariants(settingsList, IrisSettings::getIrisLectureChatSettings) : null;
+        var selectedVariant = !minimal ? getCombinedSelectedVariant(settingsList, IrisSettings::getIrisLectureChatSettings) : null;
```

Comment:
I do not think that this is a good usage of `var`

```suggestion
        SortedSet<String> allowedVariants = !minimal ? getCombinedAllowedVariants(settingsList, IrisSettings::getIrisLectureChatSettings) : null;
        String selectedVariant = !minimal ? getCombinedSelectedVariant(settingsList, IrisSettings::getIrisLectureChatSettings) : null;
```